veecle-telemetry/src/collector/pretty_exporter.rs

```diff
@@ -18,10 +18,15 @@ use std::prelude::v1::String;
 /// use veecle_telemetry::protocol::ExecutionId;
 ///
 /// let execution_id = ExecutionId::random(&mut rand::rng());
-/// set_exporter(execution_id, &ConsolePrettyExporter).unwrap();
+/// set_exporter(execution_id, &ConsolePrettyExporter::DEFAULT).unwrap();
 /// ```
-#[derive(Debug)]
-pub struct ConsolePrettyExporter;
+#[derive(Debug, Default)]
+pub struct ConsolePrettyExporter(());
+
+impl ConsolePrettyExporter {
+    /// A `const` version of `ConsolePrettyExporter::default()` to allow use as an `&'static`.
+    pub const DEFAULT: Self = ConsolePrettyExporter(());
+}
```

Comment:
This is different compared to the `ConsoleJsonExporter` and I'm not sure we gain a lot from it. I would prefer this to be consistent between exporters.